frontend/app/services/TouchpointBackend.scala

```diff
@@ -107,10 +107,8 @@ object TouchpointBackend {
     )
   }
 
-  val Normal = TouchpointBackend(BackendType.Default)
-  val TestUser = TouchpointBackend(BackendType.Testing)
-
-  val All = Seq(Normal, TestUser)
+  lazy val Normal = TouchpointBackend(BackendType.Default)
+  lazy val TestUser = TouchpointBackend(BackendType.Testing)
```

Comment:
I'd suggest at a minimum there should be a comment explaining why it's important that this is lazy.